### PR TITLE
Milestone v0.5.0

### PR DESCRIPTION
**Pull request by wild-endeavor**
Propeller: v0.2.64
Admin: v0.2.11
Flytekit: v0.10.0
Console: v0.5.3
IDL: 0.17.34

### CODE REVIEW

File: CHANGELOG/CHANGELOG-v0.5.0.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Flyte V0.5.0
+
+## Infrastructure
+
+## Core Platform
+
+
```
Comment (wild-endeavor) [inline CHANGELOG/CHANGELOG-v0.5.0.md:7]:
```suggestion
## Plugins
Cleaned up Presto executor to send namespace as user to improve queueing
```

File: CHANGELOG/CHANGELOG-v0.5.0.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Flyte V0.5.0
+
+## Infrastructure
+
+## Core Platform
+
+## Plugins
+Cleaned up Presto executor to send namespace as user to improve queueing
```
Comment (EngHabu) [inline CHANGELOG/CHANGELOG-v0.5.0.md:8]:
```suggestion
- Cleaned up Presto executor to send namespace as user to improve queueing
```

File: CHANGELOG/CHANGELOG-v0.5.0.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Flyte V0.5.0
+
+## Infrastructure
+
+## Core Platform
+
```
Comment (EngHabu) [inline CHANGELOG/CHANGELOG-v0.5.0.md:6]: - [Run to completion](https://flyte.readthedocs.io/en/latest/user/features/on_failure_policy.html) to enable workflows to continue executing even if one or more branches fail.

File: CHANGELOG/CHANGELOG-v0.5.0.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Flyte V0.5.0
+
+## Infrastructure
+
```
Comment (EngHabu) [inline CHANGELOG/CHANGELOG-v0.5.0.md:4]:
```suggestion
- Moved CI/CD to Github Actions
- Added end-to-end tests as part of the PR & master merges. 
- Enable CI system to run on forks.

```

File: CHANGELOG/CHANGELOG-v0.5.0.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Flyte V0.5.0
+
+## Infrastructure
+- Moved CI/CD to Github Actions
+- Added end-to-end tests as part of the PR & master merges. 
+- Enable CI system to run on forks.
+
+## Core Platform
+- [Run to completion](https://flyte.readthedocs.io/en/latest/user/features/on_failure_policy.html) to enable workflows to continue executing even if one or more branches fail.
```
Comment (EngHabu) [inline CHANGELOG/CHANGELOG-v0.5.0.md:9]:
```suggestion
- [Single Task Execution](https://flyte.readthedocs.io/en/latest/user/features/single_task_execution.html) to enable registering and launching tasks outside the scope of a workflow to enable faster iteration and a more intuitive development workflow.
- [Run to completion](https://flyte.readthedocs.io/en/latest/user/features/on_failure_policy.html) to enable workflows to continue executing even if one or more branches fail.
```

File: CHANGELOG/CHANGELOG-v0.5.0.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+# Flyte V0.5.0
```
Comment (EngHabu) [inline CHANGELOG/CHANGELOG-v0.5.0.md:1]:
```suggestion
# Flyte v0.5.0
```

**Pull request merged**